src/plugins/data/common/search/aggs/buckets/histogram.ts

```diff
@@ -127,9 +131,12 @@ export const getHistogramBucketAgg = ({
           return childSearchSource
             .fetch(options)
             .then((resp: any) => {
+              const min = get(resp, 'aggregations.minAgg.value', 0);
+              const max = get(resp, 'aggregations.maxAgg.value', 0);
```

Comment:
nit: Probably don't need lodash here
```suggestion
              const min = resp.aggregations?.minAgg?.value ?? 0;
              const max = resp.aggregations?.maxAgg?.value ?? 0;
```